egs/musdb18/X-UMX/local/conf.yml

```diff
@@ -0,0 +1,44 @@
+# Training config
+training:
+  epochs: 1000
+  batch_size: 14
+  loss_combine_sources: yes
+  loss_use_multidomain: yes
+  mix_coef: 10.0
+  val_dur: 80.0
+# Optim config
+optim:
+  optimizer: adam
+  lr: 0.001
+  patience: 1000
+  lr_decay_patience: 80
+  lr_decay_gamma: 0.3
+  weight_decay: 0.00001
+# Data config
+data:
+  train_dir: # Please insert here the path to the MUSDB18 dataset
```

Comment:
This is automatically set in the `run.sh` right? If yes, remove that comment, it's confusing.

Reply:
Yes, unless running from `run.sh`.
OK, I will remove this part.

Reply:
Resolved by #490 fa6ae66